packages/theme/pages/Product.vue

```diff
@@ -434,7 +425,7 @@ export default defineComponent({
       openTab,
       product,
       breadcrumbs,
-      reviews,
+      reviews: productReviews.value.data,
```

Comment:
default value is null for the reviews variable of useReview, so I think we should handle that case
```suggestion
      reviews: productReviews.value?.data,
```